### PR TITLE
Create a page for LSP1 Type IDs

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

In this PR we create a page that contains all the LSP1 Type IDs

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-8623 Add Type IDs list to the `docs.lukso.tech`](https://app.clickup.com/t/2645698/DEV-8623)

**Comment by Hugoo** [comment]
Task linked: [DEV-8623 Add Type IDs list to the `docs.lukso.tech`](https://app.clickup.com/t/2645698/DEV-8623)

**Comment by Hugoo** [comment]
Task linked: [DEV-8623 Add Type IDs list to the `docs.lukso.tech`](https://app.clickup.com/t/2645698/DEV-8623)

File: docs/contracts/type-ids.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: Interfaces IDs
```
Comment (CJ42) [inline docs/contracts/type-ids.md:2]:
```suggestion
title: Universal Receiver Type IDs
```

**Review by CJ42 — CHANGES_REQUESTED** [review]
Could you please add the path to this file in the `sidebar.js` file, so that this page shows in the docs.

image

**Review by skimaharvey — APPROVED** [review]

**Pull request merged**